.github/workflows/cd.yml

```diff
@@ -20,10 +20,17 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
+    - name: download heroku cli
+      uses: wei/curl@master
```

Comment:
I would avoid using Actions that almost does nothing (look here what it does: https://github.com/wei/curl/blob/master/entrypoint.sh). Why not using curl directly since it already is inside ubuntu? We had the problem with heroku because an Action was used for heroku when it was not necessary, so better avoid it when you can, especially those from unknown devs.

Reply:
2f299c3

Reply:
Can you check and copy also the curl options in that entrypoint that are useful?

Reply:
3848281